webperf-getting-started/Getting started with web performance.md

```diff
@@ -0,0 +1,184 @@
+# Getting started with web performance in 2019
+
+There's many very good articles out there how to use a new feature X or technique Y or tool Z to improve the performance of your websites. Navigating the labyrinth of modern web performance might be tough though if you're new to the topic. In this article I will try to come up with a guide to getting started your performance journey.
+
+
+# Know your users
+
+Before you start, it's good to know who are your users, and how they use your website, in order to take informed decisions later. The important things from performance aspect are for example:
```

Comment:
> it's good to know who are your users

it's good to know _who your users are_

> in order to take informed decisions

in order to _make_ informed decisions